packages/react-slider/src/components/Slider/Slider.types.ts

```diff
@@ -41,43 +41,44 @@ export type SliderSlots = {
   thumbWrapper: React.HTMLAttributes<HTMLElement> & React.RefAttributes<HTMLDivElement>;
 
   /**
-   * The draggable icon used to select a given value from the **Slider**.
+   * The draggable icon used to select a given value from the Slider.
+   * This is the element containing `role = 'slider'`.
    */
   thumb: React.HTMLAttributes<HTMLDivElement> & React.RefAttributes<HTMLDivElement>;
 
   /**
-   * The area in which the **Slider's** rail allows for the thumb to be dragged.
+   * The area in which the Slider's rail allows for the thumb to be dragged.
    */
   activeRail: React.HTMLAttributes<HTMLElement> & React.RefAttributes<HTMLDivElement>;
 };
 
 export interface SliderCommon extends Omit<React.HTMLAttributes<HTMLDivElement>, 'onChange'> {
   /**
-   * The starting value for an `uncontrolled` **Slider**.
+   * The starting value for an `uncontrolled` Slider.
```

Comment:
Small thing, "uncontrolled" and "controlled" shouldn't have code formatting since they don't correspond to prop names (bold or italic would be okay though)